src/raygun.rum/index.js

```diff
@@ -897,6 +918,13 @@ var raygunRumFactory = function(window, $, Raygun) {
     // =                                                                              =
     // ================================================================================
 
+    function resumeCollectingMetricsFromNow() {
+      if(self.stopCollectingMetrics) {
+        self.offset = window.performance.getEntries().length;
```

Comment:
what is the offset for? I've seen it a few times but never known what it means

Reply:
The provider gets performance timings via the `getEntries` method on the performance API and because that returns a list of the all timings (since the page was first loaded) it needs to know which ones have been 'tracked' and thus 'sent to raygun' or not. 

So `this.offset` (or `self.offset` depending on scoping) is the index (+1) of the last resource which was tracked.